### PR TITLE
Add README

### PR DESCRIPTION
**Pull request by preston-evans98**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+circumstances.
+
+## Getting Started
+
+### Set up Celestia
+
+Set up a Celestia light node running on the Arabica testnet
+
+1. Clone the repository: `git clone https://github.com/celestiaorg/celestia-node.git`.
+1. `cd celestia-node`
```
Comment (bkolad) [inline README.md:23]: Let's fix numbering 1,2...

File: README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+1. Checkout the code at v0.7.1: `git checkout tags/v0.7.1`
+1. Build and install the celestia binary: `make build && make go-install`
+1. Build celestia's key management tool `make cel-key`
+1. Initialize the node: `celestia light init --p2p.network arabica`
+1. Start the node with rpc on port 11111: `./celestia light start --core.ip https://limani.celestia-devops.dev --p2p.network arabica --gateway --rpc.port 11111`
```
Comment (bkolad) [inline README.md:28]: Should it be `celestia` instead of `./celestia`?

Comment (preston-evans98) [reply README.md]: Fixed!

File: README.md (hunk 1 above)
Comment (bkolad) [reply README.md:23]: > Markdown will render this as `1, 2, 3...`. This way, you don't have to re-do the numbering each time the list changes

Cool I didn't know about it

File: README.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+1. Clone the repository: `git clone https://github.com/celestiaorg/celestia-node.git`.
+1. `cd celestia-node`
+1. Checkout the code at v0.7.1: `git checkout tags/v0.7.1`
+1. Build and install the celestia binary: `make build && make go-install`
+1. Build celestia's key management tool `make cel-key`
+1. Initialize the node: `celestia light init --p2p.network arabica`
+1. Start the node with rpc on port 11111: `celestia light start --core.ip https://limani.celestia-devops.dev --p2p.network arabica --gateway --rpc.port 11111`
+1. Obtain a JWT for RPC access: `celestia light auth admin --p2p.network arabica`
+1. Copy the JWT and and save it in main.rs as `const NODE_AUTH_TOKEN`
```
Comment (bkolad) [inline README.md:30]: It is called `CELESTIA_NODE_AUTH_TOKEN` in the code

Comment (preston-evans98) [reply README.md:30]: Fixed!

**Review by bkolad — APPROVED** [review]

**Pull request merged**